client/src/components/common/ErrorScreen.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './ErrorScreen.scss';
+
+const getTitle = (title, code) => title || ({
+  404: 'Not Found',
+  403: 'Access Restricted',
+})[code];
+
+const ErrorScreen = ({ title, children, code }) => (
+  <div className="error-screen">
```

Comment:
It's a small thing but we have a convention in Lumen that the parent element of each component has the name of the component as a className - i.e. `ErrorScreen` rather than `error-sceen` - and that the stylesheet for the component is scoped to that className